atlassian_jwt_auth/contrib/django/middleware.py

```diff
@@ -1,21 +1,17 @@
 from django.conf import settings
 
-import atlassian_jwt_auth
-from ..server.helpers import _requires_asap
-from .utils import parse_jwt, verify_issuers, _build_response
-from .decorators import _get_verifier
+from atlassian_jwt_auth.frameworks.django.middleware import (
+    OldStyleASAPMiddleware
+)
 
 
-class ASAPForwardedMiddleware(object):
+class ProxiedAsapMiddleware(OldStyleASAPMiddleware):
```

Comment:
Should we make use of `django.utils.deprecation.MiddlewareMixin` ?

Reply:
Good point. I didn't think about this middleware being used by Django 1.11+ consumers. Fixed!